ivy/ivy.xml

```diff
@@ -136,6 +136,10 @@
 		<!-- RabbitMQ dependencies -->
 		<dependency org="com.rabbitmq" name="amqp-client" rev="3.6.5" conf="*->default" />
 
+		<!-- Kafka Dependencies -->
```

Comment:
@AyalCiobotaru I think these should be moved to src/plugin/indexer-kafka/build-ivy.xml